pkg/controller/replicaset/replica_set.go

```diff
@@ -485,22 +480,15 @@ func (rsc *ReplicaSetController) manageReplicas(filteredPods []*v1.Pod, rs *exte
 		for i := 0; i < diff; i++ {
 			go func() {
 				defer wg.Done()
-				var err error
-
-				if rsc.garbageCollectorEnabled {
-					var trueVar = true
-					controllerRef := &metav1.OwnerReference{
-						APIVersion: getRSKind().GroupVersion().String(),
-						Kind:       getRSKind().Kind,
-						Name:       rs.Name,
-						UID:        rs.UID,
-						Controller: &trueVar,
-					}
-					err = rsc.podControl.CreatePodsWithControllerRef(rs.Namespace, &rs.Spec.Template, rs, controllerRef)
-				} else {
-					err = rsc.podControl.CreatePods(rs.Namespace, &rs.Spec.Template, rs)
+				isController := true
+				controllerRef := &metav1.OwnerReference{
+					APIVersion: getRSKind().GroupVersion().String(),
+					Kind:       getRSKind().Kind,
+					Name:       rs.Name,
+					UID:        rs.UID,
+					Controller: &isController,
```

Comment:
Will `isController` ever be `false`? If we just want a &true, I'd suggest adding `func BoolPtr` to pkg/util/util.go and just use `BoolPtr(true)`. Or `const isController = true`.